src/api/spec/models/workflow/step/link_package_step_spec.rb

```diff
@@ -380,51 +380,6 @@
         it_behaves_like 'insufficient permission on target project'
         it_behaves_like 'insufficient permission to create new target project'
       end
-
-      context 'when scmsync is active' do
```

Comment:
Removing this context make is unnecessary to keep its cassettes under `spec/cassettes/Workflow_Step_LinkPackageStep/_call/when_the_SCM_is_GitHub/when_scmsync_is_active/`. Same thing for the other two removed contexts.

Reply:
Which other two contexts?

Reply:
Ah, the nested ones. Yeah they are gone..